actor/builtin/paymentbroker/paymentbroker.go

```diff
@@ -205,7 +207,9 @@ func (pb *Actor) CreateChannel(vmctx exec.VMContext, target address.Address, eol
 // target Redeem(200)          -> Payer: 1000, Target: 200, Channel: 800
 // target Close(500)           -> Payer: 1500, Target: 500, Channel: 0
 //
-func (pb *Actor) Redeem(vmctx exec.VMContext, payer address.Address, chid *types.ChannelID, amt *types.AttoFIL, validAt *types.BlockHeight, condition *types.Predicate, sig []byte) (uint8, error) {
+// If a condition is provided in the voucher, concatenate its params with supplied params to send a message.
```

Comment:
I think this comment is trying to explain that internally the Redeem function will send a message that includes the combination of redeemer supplied parameters and the payer supplied parameters within the input condition.  I think putting this comment about internals is misleading, at least without painstakingly explaining the condition checking message call which feels out of scope of this comment, and should be moved down to the `checkCondition` call.

`redeemerSuppliedParams` might be better called `redeemerWitness` (or `redeemerProof` but proof is overloaded everywhere) because params is too general and confusing because these params are getting passed into the `Redeem` function as params but the intent is that they are params of the condition checking internal message call.  If we really want to keep params we could be more specific: `redeemerConditionCheckingParams`